sdk/nodejs/tests/deasync.spec.ts

```diff
@@ -0,0 +1,156 @@
+// Copyright 2016-2018, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// tslint:disable
+
+import * as assert from "assert";
+import { output, Output, Resource } from "../index";
+import { asyncTest } from "./util";
+import { promiseResult, liftProperties } from "../utils";
+
+// function test(val: any, expected: any) {
```

Comment:
Should we remove the commented-out tests?

Reply:
yes

Reply:
you're too fast :)